must-gather/Centos-Base.repo

```diff
@@ -0,0 +1,6 @@
+[centos]
+name=CentOS-7
```

Comment:
This tool is completely linked to RH. Why not to use an UBI image?

Reply:
Looks like this is not required at all.  I will remove it.